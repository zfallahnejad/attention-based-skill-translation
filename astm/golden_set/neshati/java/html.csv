157882,html
418556,html
1622894,html
571407,html
260990,html
573032,html
131872,html
478399,html
871026,html
1393766,html
157247,html
992484,html
1927832,html
268273,html
