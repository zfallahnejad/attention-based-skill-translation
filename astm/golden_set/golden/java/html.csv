157882,105,77,0.5769230769230769
418556,62,88,0.41333333333333333
1622894,29,24,0.5471698113207547
571407,27,25,0.5192307692307693
260990,26,30,0.4642857142857143
573032,22,18,0.55
131872,18,18,0.5
871026,14,11,0.56
478399,14,16,0.4666666666666667
1393766,13,7,0.65
1927832,12,13,0.48
992484,12,4,0.75
157247,12,6,0.6666666666666666
268273,11,6,0.6470588235294118
1447173,10,3,0.7692307692307693
1138559,10,6,0.625
1654265,10,5,0.6666666666666666
